src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24741.cs

```diff
@@ -0,0 +1,46 @@
+﻿#if WINDOWS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue24741 : _IssuesUITest
+	{
+		public Issue24741(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "Unable to select tab after backing out of page and returning";
+
+		[Test]
+		[Category(UITestCategories.TabbedPage)]
+		public void SelectTabAfterNavigation()
```

Comment:
Updated test, simplified. Now is easy to wait or tap elements using both, AutomationId or with the view text.
![image](https://github.com/user-attachments/assets/76264c81-8478-4eba-af92-912b34a7a3cd)

Reply:
@jsuarezruiz, Thank you for your effort in updating the test case for this complex scenario.